NPCDrops.cs

```diff
@@ -19,8 +19,8 @@ public class NPCDrops : GlobalNPC
 			ItemID.RangerEmblem,
 			ItemID.SorcererEmblem,
 			ItemID.SummonerEmblem,
-			ModLoader.GetMod("Tremor").ItemType("AlchemistEmblem"),
-			ModLoader.GetMod("Tremor").ItemType("ThrowerEmblem")
+			Tremor.instance.ItemType("AlchemistEmblem"),
+			Tremor.instance.ItemType("ThrowerEmblem")
```

Comment:
Unfortunately, this throws 

> Exception has been thrown by the target of an invocation.
>    at System.RuntimeTypeHandle.CreateInstance

It will need to be changed back